conf/layer.conf

```diff
@@ -0,0 +1,13 @@
+# We have a conf and classes directory, add to BBPATH
+BBPATH .= ":${LAYERDIR}"
+
+# We have recipes-* directories, add to BBFILES
+BBFILES += "${LAYERDIR}/recipes-*/*/*.bb \
+            ${LAYERDIR}/recipes-*/*/*.bbappend"
+
+BBFILE_COLLECTIONS += "meta-smartrack"
+BBFILE_PATTERN_meta-smartrack = "^${LAYERDIR}/"
+BBFILE_PRIORITY_meta-smartrack = "24"
```

Comment:
Can you place a brief comment here on how to come out this magic number (24)? It will help for future references and maintenance.

Reply:
Added a comment above the line. I also changed priority from 24 to 25. My original intention was to allow meta-smartrack have priority over other Toradex layers. Layer priority 24 (assumed by meta-toradex-demos) was the highest among Toradex layers, so to achieve the aim I went one value higher.